src/app/clusters/wifi-network-management-server/wifi-network-management-server.cpp

```diff
@@ -0,0 +1,169 @@
+/**
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "wifi-network-management-server.h"
+
+#include <app/AttributeAccessInterfaceRegistry.h>
+#include <app/InteractionModelEngine.h>
+#include <app/reporting/reporting.h>
+#include <app/util/config.h>
+#include <lib/core/Global.h>
+#include <lib/support/CodeUtils.h>
+
+#include <algorithm>
+#include <cctype>
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::WiFiNetworkManagement::Attributes;
+using namespace std::placeholders;
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace WiFiNetworkManagement {
+
+Global<Server> gServerInstance;
```

Comment:
should this be in an anonymous namespace so that access can only happen via the Intance call?